src/specular/stages/batcher_transactions.rs

```diff
@@ -0,0 +1,101 @@
+use std::sync::mpsc;
+
+use ethers::{
+    abi::parse_abi_str,
+    contract::Lazy,
+    prelude::BaseContract,
+    types::{Bytes, Selector, U256},
+};
+use eyre::Result;
+use std::collections::VecDeque;
+
+use crate::derive::stages::batcher_transactions::BatcherTransactionMessage;
+use crate::derive::PurgeableIterator;
+
+pub struct SpecularBatcherTransactions {
```

Comment:
```suggestion
pub struct BatcherTransactions {
```
imo ^ is fine since we're in a `specular` submodule; same for other structs (`SpecularBatcherTransaction`, `SpecularBatches` etc) -- and just qualify w/ `specular::` to disambiguate when necessary